osu.Game/Skinning/Editor/SkinSelectionHandler.cs

```diff
@@ -151,11 +151,26 @@ public override bool HandleMovement(MoveSelectionEvent<ISkinnableDrawable> moveE
             {
                 Drawable drawable = (Drawable)c.Item;
                 drawable.Position += drawable.ScreenSpaceDeltaToParentSpace(moveEvent.ScreenSpaceDelta);
+
+                updateDrawableAnchorIfUsingClosest(c.Item);
             }
 
             return true;
         }
 
+        private void updateDrawableAnchorIfUsingClosest(ISkinnableDrawable item)
```

Comment:
The way these anchor application methods are written is really confusing to me. There are several code paths that do slightly different things. This and `applyCustomAnchor` should really be one method.

Reply:
I think `UsesFixedAnchor` is also fine, fwiw. As long as `Overrides` isn't in the name, since that naming is also a bit backwards (as it specifies which behaviour *is turned off*, and not which one *is actually used*).

Reply:
Ah, okay. I was conceiving the `Overrides` as "a thing the user did to the object." I'll go with `UsesFixedAnchor`, then.